paddle/operators/lrn_op.cc

```diff
@@ -19,6 +19,108 @@ namespace operators {
 
 using framework::Tensor;
 
+template <typename T>
+struct LRNFunctor<platform::CPUPlace, T> {
+  void operator()(const framework::ExecutionContext& ctx,
+                  const framework::Tensor* input, framework::Tensor* out,
+                  framework::Tensor* mid, int N, int C, int H, int W, int n,
+                  T k, T alpha, T beta) {
+    auto x_v = framework::EigenVector<T>::Flatten(*input);
+
+    const int start = -(n - 1) / 2;
+    const int end = start + n;
+
+    auto e_mid = framework::EigenTensor<T, 4>::From(*mid);
+    e_mid.device(ctx.GetEigenDevice<platform::CPUPlace>()) = e_mid.constant(k);
```

Comment:
For the CPU implementation of Eigen, there is no need to use `.device()`.

```cpp
e_mid.setConstant(k);
```

Reply:
Done!
Thanks!